cmd/juju/cloud/updatecredential.go

```diff
@@ -369,6 +362,36 @@ func (c *updateCredentialCommand) updateRemoteCredentials(ctx *cmd.Context, upda
 		ctx.Warningf("Could not update credentials remotely, on controller %q", controllerName)
 		erred = cmd.ErrSilent
 	}
+	if err := processUpdateCredentialResult(ctx, accountDetails, "updat", results); err != nil {
```

Comment:
I'd just pass through the verb for printing, rather than "updat" it should just be "updated"